Annonce d'informations pour le semestre de mobilité 2022

gtedavid opened the pull request:
File en attente de remplissage - attente de la réception du contenu par l'auteur

gtedavid committed:
File creation - future announcement on abroad semester

File en attente de remplissage - attente de la réception du contenu par l'auteur

gtedavid committed:
First draft for the announcement

@melisandeGB Si tu prux dire ton avis sur cette première partie, le mieux c'est de mettre en draft pull request pour le moment

Je vais voir si quelqu'un peut résumer les options des destinations...
@Kadiatou-yatt ?

gtedavid committed:
update formate

noftaly commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,10 @@
+Hello les L2 ||<@&794628669737009183>|| !
+
+Avec la publication des classements des 3 premiers semestres et le choix de la mobilité qui arrive à grand pas,
+J'ai créé un formulaire qui va permettre à tout le monde de se pré-situer (ça reste anonyme, ne vous inquiétez pas),
```
```suggestion
[Un formulaire circule, permettant à tout le monde de se pré-situer (ça reste anonyme, ne vous inquiétez pas)](https://forms.gle/5oC6osJFc3EXDz1H6)
```

noftaly commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,10 @@
+Partagez le au max comme ça on peut tous avoir une idée d'où on peut se retrouver l'an prochain 🙂
+
+Merci!
+(Lien du form)[https://forms.gle/JU7Bc7TcRWAAFZuRA]
```
```suggestion
```

noftaly commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,10 @@
+
+Avec la publication des classements des 3 premiers semestres et le choix de la mobilité qui arrive à grand pas,
+J'ai créé un formulaire qui va permettre à tout le monde de se pré-situer (ça reste anonyme, ne vous inquiétez pas),
+Les résultats sont disponibles après avoir submit le form.
```
```suggestion
Les résultats sont disponibles après avoir l'avoir rempli.
```

noftaly reviewed (changes requested):
Modifié pour que ce soit + propre, et j'ai mis le lien du formulaire qui existait déjà

gtedavid committed:
Update resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md

Co-authored-by: Elliot <elliot.maisl@gmail.com>

gtedavid committed:
Update resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md

Co-authored-by: Elliot <elliot.maisl@gmail.com>

gtedavid committed:
Update resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md

Co-authored-by: Elliot <elliot.maisl@gmail.com>

gtedavid committed:
Update Destinations & creator

!!!!! DESTINATIONS URL IMPLEMENTATION FALSE !!!!

gtedavid committed:
Supp numbers

Supp numbers old survey

gtedavid committed:
Update format and warning

updated link formatting and warning info

gtedavid committed:
Last link format update

Updated link on bottom message

gtedavid committed:
Extra word DEL + ème->e

Places extra word deleted

ème correction to e

noftaly commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,82 @@
+Ces informations vous permettront de les comparer plus facilement :map:
+N'hésitez pas à poser vos **questions relatives aux destinations sur <#889180185574924328>** (ceux qui ont déjà voyagé dans vos futures destinations pourront vous y répondre) !
+**:warning: TOUTES LES DONNÉES DE COÛTS ET DE CLASSEMENT DATENT DE SEPTEMBRE 2021 :warning:
+POUR TOUTES LES DONNÉES MANQUANTES, à CE JOUR NOUS NE SAVONS PAS SI NOUS LES AVONS**
```
```suggestion
POUR TOUTES LES DONNÉES MANQUANTES, À CE JOUR NOUS NE SAVONS PAS SI NOUS LES AVONS**
```

noftaly commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,82 @@
+
+Ces informations vous permettront de les comparer plus facilement :map:
+N'hésitez pas à poser vos **questions relatives aux destinations sur <#889180185574924328>** (ceux qui ont déjà voyagé dans vos futures destinations pourront vous y répondre) !
+**:warning: TOUTES LES DONNÉES DE COÛTS ET DE CLASSEMENT DATENT DE SEPTEMBRE 2021 :warning:
```
Pourquoi tout en majuscule ? c'est déjà en gras et avec un ⚠️ 😄

gtedavid replied:
jsp, je pensais que c'était important de le souligner mais on peut enlever le gras 😅

ou jsp trop 😅

noftaly replied:
enlever les majuscules et mettre une ligne avant ca suffira je pense, ca le mettra déjà bien en valeur avec le ⚠️ et le gras

noftaly commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,82 @@
+**:warning: TOUTES LES DONNÉES DE COÛTS ET DE CLASSEMENT DATENT DE SEPTEMBRE 2021 :warning:
+POUR TOUTES LES DONNÉES MANQUANTES, à CE JOUR NOUS NE SAVONS PAS SI NOUS LES AVONS**
+
+**Choix final le  [LIEN A VENIR]**
```
```suggestion
**Choix final le  [LIEN A VENIR]**
```
il manque le lien

gtedavid replied:
Ik je sais pas 
et j'ai pas d'info 

mais on peut l'enlever (for now)
A moins qu'ils vous ont donné le sondage moodle où il faudra faire le choix

noftaly replied:
non on a rien pour le moment, autant l'enlever alors je pense 😄

noftaly commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,82 @@
+__Nombre de places__ : **30-60**
+
+:flag_ca:  **CANADA (MONTRÉAL) - [CU](https://www.concordia.ca/)**
+__Coût de base:__ **5450€** = mutuelles (France + Canada), aller-retour, permis, études, 2565€ de logement
```
"permis, études" ? ca veut dire quoi ?

gtedavid replied:
jsp, c'est Ivan 😅
j'ai juste copié collé

noftaly commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,82 @@
+__Coût de la vie:__ **3400-3700€** = frais + 40€/semaine extra (= 2 sorties)
+__Classement mondial:__ **1000-1400e** - __GPA requis:__ GPA >= 2.0
+__Nombre de places__ : **36-48 / 54-60**
+|| ||
```
|| || ?

gtedavid replied:
c'est un reste on peut l'enlever

noftaly replied:
càd ? 

jte laisse l'enlever alors

gtedavid replied:
oups
'jai commit une version où je l'ai enlevé 😅
tu commentes sur une version outdated du coup

noftaly commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,82 @@
+
+**NOTES:**
+
+A titre de comparaison, **l’Efrei est placée en 2300+ e  position** dans les classements mondiaux
```
```suggestion
A titre de comparaison, **l’Efrei est placée en 2300+e  position** dans les classements mondiaux.
```

noftaly reviewed (changes requested):
Ça + remplir les "NO DATA" 😄

gtedavid committed:
Letter cap error

Line 17 À instead of à

Co-authored-by: Elliot <elliot.maisl@gmail.com>

gtedavid committed:
Update resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md

Co-authored-by: Elliot <elliot.maisl@gmail.com>

gtedavid committed:
Update ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md

Link at end updated + correction of taking out || ||

noftaly commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,81 @@
+Le coût de la vie est **le coût pour tout le voyage**, pas le coût mensuel
+
+[Source coût de la vie, coût hors-logement & hors dépenses persos/risques](https://www.numbeo.com)
+Sources classements: https://webometrics.info/en - https://www.topuniversities.com - https://cwur.org/2021-22.php
```
```suggestion
Sources classements: [Webometrics](https://webometrics.info/en) - [TopUniversities ](https://www.topuniversities.com) - [CWUR](https://cwur.org/2021-22.php)
```

gtedavid replied:
att je vais vérifier si le lien de 2022-2023 existe

gtedavid replied:
nope

noftaly commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,81 @@
+[Source coût de la vie, coût hors-logement & hors dépenses persos/risques](https://www.numbeo.com)
+Sources classements: https://webometrics.info/en - https://www.topuniversities.com - https://cwur.org/2021-22.php
+
+
+[Toutes les infos](https://moodle.myefrei.fr/pluginfile.php/229799/mod_resource/content/2/Amphi%20L3%20Fall%202022.pdf)
```
```suggestion
[Toutes les infos sur Moodle](https://moodle.myefrei.fr/pluginfile.php/229799/mod_resource/content/2/Amphi%20L3%20Fall%202022.pdf)
```

gtedavid committed:
UPDATE NAME Choix

choi à fair X -> les destinations

gtedavid committed:
Update resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md

Co-authored-by: Elliot <elliot.maisl@gmail.com>

gtedavid committed:
Update resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md

Co-authored-by: Elliot <elliot.maisl@gmail.com>

noftaly commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,80 @@
+
+
+
+:flag_ca:  **CANADA (TORONTO) - [ILAC](https://www.ilac.com/toronto/)** 
```
```suggestion
**
**:flag_ca:  **CANADA (TORONTO) - [ILAC](https://www.ilac.com/toronto/)** 
```
proposition pour que ca saut une ligne même au début de message, de manière transparente :smile:

gtedavid replied:
je crois qu'il y a plutôt un pb de `**`
y en a un en trop

noftaly replied:
non c'était bon. je le refais car c'était sur un fichier outdated

gtedavid committed:
Update warning no caps

Enlever caps du msg warning

noftaly commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,80 @@
+__Classement mondial:__ **1000-1400e** - __GPA requis:__ GPA >= 2.0
+__Nombre de places__ : **36-48 / 54-60**
+
+
+
+:flag_ca:  **CANADA (TORONTO) - [ILAC](https://www.ilac.com/toronto/)**
+
+CURRENTLY NO DATA AVAILABLE
+__Nombre de places__ : **30-40**
```
```suggestion


**
**:flag_ca:  **CANADA (TORONTO) - [ILAC](https://www.ilac.com/toronto/)**
CURRENTLY NO DATA AVAILABLE
__Nombre de places__ : **30-40**
```

noftaly committed:
Update resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md

noftaly commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,80 @@
+
+**
+**:flag_ca:  **CANADA (TORONTO) - [ILAC](https://www.ilac.com/toronto/)**
+CURRENTLY NO DATA AVAILABLE
```
```suggestion
*Données pas encore compilées*
```

noftaly commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,80 @@
+__Nombre de places__ : **30-40**
+
+:flag_hu:  **HONGRIE (BUDAPEST) - [ESSCA](https://www.essca.fr/lessca/les-sites-du-groupe/essca-budapest)**
+CURRENTLY NO DATA AVAILABLE
```
```suggestion
*Données pas encore compilées*
```

noftaly commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,80 @@
+__Nombre de places__ : **30-35**
+
+:flag_cz:  **RÉPUBLIQUE TCHÈQUE (OSTRAVA) - [VSB Technical University of Ostrava](https://www.vsb.cz/en/)**
+CURRENTLY NO DATA AVAILABLE
```
```suggestion
*Données pas encore compilées*
```

noftaly commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,80 @@
+__Nombre de places__ : **18-20**
+
+:flag_za: **SOUTH AFRICA (LE CAP) - [Cape Peninsula University of Technology](https://www.cput.ac.za/)**
+CURRENTLY NO DATA AVAILABLE
```
```suggestion
*Données pas encore compilées*
```

noftaly committed:
Apply suggestions from code review

noftaly commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,80 @@
+
+Hello les L2 ||<@&794628669737009183>|| !
```
```suggestion
Hello les L2 ||<@&794628669737009183>|| !
```

noftaly commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,80 @@
+Merci!
+
+
+**__INFOS POUR LES L3: SEMESTRE INTERNATIONAL S5__**
```
```suggestion
**
** **__INFOS POUR LES L3: SEMESTRE INTERNATIONAL S5__**
```

noftaly commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,80 @@
+__Nombre de places__ : **30-40**
+
+
+**NOTES:**
```
```suggestion
**
** **NOTES:**
```

noftaly commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,80 @@
+*Données pas encore compilées*
+__Nombre de places__ : **18-20**
+
+:flag_za: **SOUTH AFRICA (LE CAP) - [Cape Peninsula University of Technology](https://www.cput.ac.za/)**
```
```suggestion
:flag_za:  **SOUTH AFRICA (LE CAP) - [Cape Peninsula University of Technology](https://www.cput.ac.za/)**
```

noftaly commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,80 @@
+__Classement mondial:__ **600-800e** - __GPA requis:__ GPA >= 1.0
+__Nombre de places__ : **35-40 / 50-51 / 59-60**
+
+:flag_gb: **ANGLETERRE (STOKE-ON-TRENT) - [SU](https://www.staffs.ac.uk/)**
```
```suggestion
:flag_gb:  **ANGLETERRE (STOKE-ON-TRENT) - [SU](https://www.staffs.ac.uk/)**
```

noftaly commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,80 @@
+__Classement mondial:__ **2000+e** - __GPA requis:__ Pas de minimum
+__Nombre de places__ : **30-60**
+
+:flag_my: **MALAISIE (KUALA LUMPUR) - [APU](https://www.apu.edu.my/)**
```
```suggestion
:flag_my:  **MALAISIE (KUALA LUMPUR) - [APU](https://www.apu.edu.my/)**
```

noftaly commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,80 @@
+__Coût de base:__ **1480€** = mutuelle France, aller-retour, 900€ de logement
+__Coût de la vie:__ **1700-2650€** = frais + 30€/semaine extra (= 2 sorties)
+__Classement mondial:__ **600-800e** - __GPA requis:__ GPA >= 1.0
+__Nombre de places__ : **35-40 / 50-51 / 59-60**
```
```suggestion
__Nombre de places:__ **35-40 / 50-51 / 59-60**
```

noftaly commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,80 @@
+__Coût de base:__ **2950€** = mutuelle France, aller-retour, 2500€ de logement
+__Coût de la vie:__ **3200-3800€** = frais + 40€/semaine extra (= 2 sorties)
+__Classement mondial:__ **2000+e** - __GPA requis:__ Pas de minimum
+__Nombre de places__ : **30-60**
```
```suggestion
__Nombre de places:__ **30-60**
```

noftaly commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,80 @@
+__Coût de base:__ **3520€** = mutuelles (France + Malaise), aller-retour, visa, 1100€ de logement
+__Coût de la vie:__ **1050-2100€** = frais + 20€/semaine extra (= 2 sorties)
+__Classement mondial:__ **2000+e** - __GPA requis:__ GPA >= 1.0
+__Nombre de places__ : **30-60**
```
```suggestion
__Nombre de places:__ **30-60**
```

noftaly commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,80 @@
+__Coût de base:__ **5450€** = mutuelles (France + Canada), aller-retour, permis, études, 2565€ de logement
+__Coût de la vie:__ **3400-3700€** = frais + 40€/semaine extra (= 2 sorties)
+__Classement mondial:__ **1000-1400e** - __GPA requis:__ GPA >= 2.0
+__Nombre de places__ : **36-48 / 54-60**
```
```suggestion
__Nombre de places:__ **36-48 / 54-60**
```

noftaly commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,80 @@
+**
+**:flag_ca:  **CANADA (TORONTO) - [ILAC](https://www.ilac.com/toronto/)**
+*Données pas encore compilées*
+__Nombre de places__ : **30-40**
+
+:flag_us:  **USA (IRVINE) - [UCI](https://uci.edu/)**
+__Coût de base:__ **7100€** = mutuelle France, aller-retour, permis, études, visa, 5200€ de logement
++ Preuve de 10 000 € de ressources financières
+__Coût de la vie:__ **4100-4600€** = frais + 50€/semaine extra (= 2 sorties)
+__Classement mondial:__ **50-250e** - __GPA requis:__ GPA >= 2.0
+__Nombre de places__ : **30-40**
+
+:flag_hu:  **HONGRIE (BUDAPEST) - [ESSCA](https://www.essca.fr/lessca/les-sites-du-groupe/essca-budapest)**
+*Données pas encore compilées*
+__Nombre de places__ : **30-35**
+
+:flag_cz:  **RÉPUBLIQUE TCHÈQUE (OSTRAVA) - [VSB Technical University of Ostrava](https://www.vsb.cz/en/)**
+*Données pas encore compilées*
+__Nombre de places__ : **18-20**
```
```suggestion
__Nombre de places:__ **30-40**

:flag_us:  **USA (IRVINE) - [UCI](https://uci.edu/)**
__Coût de base:__ **7100€** = mutuelle France, aller-retour, permis, études, visa, 5200€ de logement
+ Preuve de 10 000 € de ressources financières
__Coût de la vie:__ **4100-4600€** = frais + 50€/semaine extra (= 2 sorties)
__Classement mondial:__ **50-250e** - __GPA requis:__ GPA >= 2.0
__Nombre de places:__ **30-40**

:flag_hu:  **HONGRIE (BUDAPEST) - [ESSCA](https://www.essca.fr/lessca/les-sites-du-groupe/essca-budapest)**
*Données pas encore compilées*
__Nombre de places:__ **30-35**

:flag_cz:  **RÉPUBLIQUE TCHÈQUE (OSTRAVA) - [VSB Technical University of Ostrava](https://www.vsb.cz/en/)**
*Données pas encore compilées*
__Nombre de places:__ **18-20**
```

noftaly committed:
Apply suggestions from code review

melisandeGB committed:
Update ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md

gtedavid commented:
~~ANNOUNCEMENT WILL BE POSTED TOMORROW MORNING~~

gtedavid commented:
POSTPONED to this Week-end !!!!
en attente de données d'Esteban

gtedavid committed:
Ajout des données - Annonce à modifier

Data calculée par Esteban ajoutée

Annonce doit être modifier vu que le form est fermé

gtedavid committed:
Début de l'adaptation des données

Je garde les deux
ça donnera une idée de fourchette
@iv-stpn @noftaly 

(début de l'annonce encore à modifier)

noftaly committed:
Normalize information format and remove old info

noftaly committed:
fix link

gtedavid committed:
Correction placements et pluriel de places

Ordre de l'affichage des infos (nombre de places, n'était pas placé de la même façon pour toutes les destinations

gtedavid committed:
MàJ de la première partie de l'annonce

Changement de la première partie de l'annonce

gtedavid commented:
@melisandeGB Tu peux relire stp, l'annonce a à un peu changé

(@noftaly je t'ai request une review juste pour que tu vois les changements que j'ai pu faire)

melisandeGB commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,97 @@
+
+Ces informations vous permettront de les comparer plus facilement :map:
+N'hésitez pas à poser vos **questions relatives aux destinations sur <#889180185574924328>** ou dans les channels des destinations. Ceux qui ont déjà voyagé dans vos futures destinations pourront vous y répondre !
+**:warning: Toutes les données monétaires sont des estimations, suivants vos habitudes ou trajets, elles sont susceptibles de varier  :warning:**
```
```suggestion
**:warning: Toutes les données monétaires sont des estimations, suivant vos habitudes ou trajets, elles sont susceptibles de varier  :warning:**
```

noftaly commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,97 @@
+Hello les L2 ||<@&794628669737009183>|| !
+
+Le form a été fermé.
+Cette annonce arrive avec un peu de retard.
```
Supprimer ce message, on ne sait pas de quel form tu parles ca va juste embrouiller les gens ^^

gtedavid replied:
donc tout simplement supprimer les 2 premières lignes

gtedavid replied:
Done ✔️

noftaly commented on resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md:
```diff
@@ -0,0 +1,97 @@
+__Nombre de places :__ 36-48 OU 54-60
+__Prix des billets d'avion aller retour :__ ~500€
+__Logement à trouver :__ Non
+
+
+
+**
+**:flag_ca:  **CANADA (TORONTO) - [ILAC](https://www.ilac.com/toronto/)**
+__Estimation du coût mensuel pour une personne :__ [935€](https://www.numbeo.com/cost-of-living/in/Toronto)
+__Classement :__
+__Nombre de places :__ 30-40
+__Prix des billets d'avion aller retour :__ ~600€
+__Logement à trouver :__ Oui
+
+
+:flag_us:  **USA (IRVINE) - [UCI](https://uci.edu/)**
+__Estimation du coût mensuel pour une personne :__ [915€](https://www.numbeo.com/cost-of-living/in/Irvine)
+__Classement :__ 232
+__Nombre de places :__ 30-40
+__Prix des billets d'avion aller retour :_ ~1000€
+__Logement à trouver :__ Oui
+
+
+:flag_hu:  **HONGRIE (BUDAPEST) - [ESSCA](https://www.essca.fr/lessca/les-sites-du-groupe/essca-budapest)**
+__Prix des billets d'avion aller retour :__ ~220€
+__Estimation du coût mensuel pour une personne :__ [520€](https://www.numbeo.com/cost-of-living/in/Budapest)
+__Classement :__
+__Nombre de places :__ 30-35
+__Logement à trouver :__ Oui 
+
+
+:flag_cz:  **RÉPUBLIQUE TCHÈQUE (OSTRAVA) - [VSB](https://www.vsb.cz/en/)**
+__Estimation du coût mensuel pour une personne :__ [690€](https://www.numbeo.com/cost-of-living/in/Ostrava)
+__Classement :__ 1001-1200
+__Nombre de places :__ 18-20
+__Prix des billets d'avion aller retour :__ ~250€
+__Logement à trouver :__ Non
+
+
+:flag_za:  **SOUTH AFRICA (LE CAP) - [CPUT](https://www.cput.ac.za/)**
```
```suggestion

:flag_gb:  **ANGLETERRE (STOKE-ON-TRENT) - [SU](https://www.staffs.ac.uk/)**
__Estimation du coût mensuel pour une personne :__ [745€](https://www.numbeo.com/cost-of-living/in/Stoke-On-Trent)
__Classement :__
__Nombre de places :__ 30-60
__Prix des billets d'avion aller retour :__ ~175€
__Logement à trouver :__ Non 

:flag_my:  **MALAISIE (KUALA LUMPUR) - [APU](https://www.apu.edu.my/)**
__Estimation du coût mensuel pour une personne :__ [455€](https://www.numbeo.com/cost-of-living/in/Kuala-Lumpur)
__Classement :__ 275 d'Asie
__Nombre de places :__ 30-60
__Prix des billets d'avion aller retour :__ ~800€
__Logement à trouver :__ Non

:flag_ca:  **CANADA (MONTRÉAL) - [CU](https://www.concordia.ca/)**
__Estimation du coût mensuel pour une personne :__ [835€](https://www.numbeo.com/cost-of-living/in/Montreal)
__Classement :__ 525
__Nombre de places :__ 36-48 OU 54-60
__Prix des billets d'avion aller retour :__ ~500€
__Logement à trouver :__ Non


**
**:flag_ca:  **CANADA (TORONTO) - [ILAC](https://www.ilac.com/toronto/)**
__Estimation du coût mensuel pour une personne :__ [935€](https://www.numbeo.com/cost-of-living/in/Toronto)
__Classement :__
__Nombre de places :__ 30-40
__Prix des billets d'avion aller retour :__ ~600€
__Logement à trouver :__ Oui

:flag_us:  **USA (IRVINE) - [UCI](https://uci.edu/)**
__Estimation du coût mensuel pour une personne :__ [915€](https://www.numbeo.com/cost-of-living/in/Irvine)
__Classement :__ 232
__Nombre de places :__ 30-40
__Prix des billets d'avion aller retour :_ ~1000€
__Logement à trouver :__ Oui

:flag_hu:  **HONGRIE (BUDAPEST) - [ESSCA](https://www.essca.fr/lessca/les-sites-du-groupe/essca-budapest)**
__Prix des billets d'avion aller retour :__ ~220€
__Estimation du coût mensuel pour une personne :__ [520€](https://www.numbeo.com/cost-of-living/in/Budapest)
__Classement :__
__Nombre de places :__ 30-35
__Logement à trouver :__ Oui 

:flag_cz:  **RÉPUBLIQUE TCHÈQUE (OSTRAVA) - [VSB](https://www.vsb.cz/en/)**
__Estimation du coût mensuel pour une personne :__ [690€](https://www.numbeo.com/cost-of-living/in/Ostrava)
__Classement :__ 1001-1200
__Nombre de places :__ 18-20
__Prix des billets d'avion aller retour :__ ~250€
__Logement à trouver :__ Non

:flag_za:  **SOUTH AFRICA (LE CAP) - [CPUT](https://www.cput.ac.za/)**
```
2 lignes vides = nouveau message

gtedavid replied:
ah ok
je vais continuer à réorganiser les éléments pt-être

gtedavid committed:
Update resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md

Co-authored-by: Mélisande  <72623143+melisandeGB@users.noreply.github.com>

gtedavid committed:
Update resources/ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md

Co-authored-by: Elliot <elliot.maisl@gmail.com>

gtedavid committed:
Déplacement du nombre de places pour 1 destination

Déplacement de la ligne du nombre de places pour la Pologne

gtedavid committed:
Correction format soulignement

Correction souligné du 1000 des US

gtedavid committed:
Ajout saut de ligne

Saut de ligne avant Angleterre

gtedavid committed:
Correction et MàJ de l'intro

Correction sur les destinataires de l'annonce et du message d'intro

gtedavid committed:
MàJ ces classements

MàJ des classements manquants

gtedavid committed:
Classement manquant

Classement manquant pour Toronto

noftaly committed:
Rename ANNOUNCEMENT_2025_SONDAGE_SEMESTRE_D_ECHANGE_L3_2022_2023.md to ANNOUNCEMENT_2025_INFORMATIONS_MOBILITÉ_S5.md
